### DISCUSSION TITLE
Money help

### DISCUSSION THREAD

**Discussion by jerseyrain79** [moneymanagerex/moneymanagerex#6101] [Q&A]
Looking for info or help on how to store money in a bank or where ever with having to report it to irs or explain where it's from

**Comment by tactilis** [comment]
It's not clear what the problem is.

You create Deposit transactions in MMEX showing when the money is deposited in the bank. You enter the appropriate info in the Payee field to indicate who it is from. You Print the transaction list and give it to the IRS.

Taken at face value, your question has nothing really to do with MMEX.

Could you explain more clearly what the problem is and what you are actually trying to achieve?
